Sharan garcha mfst/send notification to smba for indirect membership update

SharanGarcha-MSFT opened the pull request.

SharanGarcha-MSFT committed:
Create Bot-updates-shared-channels.md

Content for bot updates for indirect members

SharanGarcha-MSFT committed:
Update Bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update Bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update Bot-updates-shared-channels.md

updated content

SharanGarcha-MSFT committed:
Update Bot-updates-shared-channels.md

updated content for membership changes

SharanGarcha-MSFT committed:
Update Bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Rename Bot-updates-shared-channels.md to bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Revise section titles

SharanGarcha-MSFT committed:
Fix formatting issue in bot updates documentation

SharanGarcha-MSFT committed:
Update Bot-updates-shared-channels.md

AjayJ12-MSFT commented on msteams-platform/concepts/build-and-test/bot-updates-shared-channels.md:
````diff
@@ -0,0 +1,204 @@
+}
+```
+
+### Membership Changes 
````
Update the title to Shared with Team events

SharanGarcha-MSFT committed:
Fix punctuation and enhance logging in shared channels bot

Corrected punctuation and improved logging messages in bot update handling for shared channels.

SharanGarcha-MSFT committed:
Merge branch 'SharanGarcha-MFST/Send-notification-to-SMBA-for-indirect-membership-update' of https://github.com/MicrosoftDocs/msteams-docs into SharanGarcha-MFST/Send-notification-to-SMBA-for-indirect-membership-update

SharanGarcha-MSFT committed:
Merge branch 'SharanGarcha-MFST/Send-notification-to-SMBA-for-indirect-membership-update' of https://github.com/MicrosoftDocs/msteams-docs into SharanGarcha-MFST/Send-notification-to-SMBA-for-indirect-membership-update

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

added content

SharanGarcha-MSFT committed:
Delete Bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

SharanGarcha-MSFT committed:
Update bot-updates-shared-channels.md

ChetanSharma-msft commented:
These changes are already incorporated as part of another PR here: https://github.com/MicrosoftDocs/msteams-docs/pull/13206

So, we are closing this PR.
